pinot-core/src/main/java/org/apache/pinot/core/query/reduce/BrokerReduceService.java

```diff
@@ -55,15 +56,12 @@
 @ThreadSafe
 public class BrokerReduceService {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(BrokerReduceService.class);
-
-  // brw -> Shorthand for broker reduce worker threads.
-  private static final String REDUCE_THREAD_NAME_FORMAT = "brw-%d";
-
   // Set the reducer priority higher than NORM but lower than MAX, because if a query is complete
   // we want to deserialize and return response as soon. This is the same as server side 'pqr' threads.
   protected static final int QUERY_RUNNER_THREAD_PRIORITY = 7;
-
+  private static final Logger LOGGER = LoggerFactory.getLogger(BrokerReduceService.class);
```

Comment:
Nit: it'd be good to keep these static constants on the top.

Reply:
This change is performed by Intellj reformat, so I's like to not change it back.